package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "o1js",
   "description": "TypeScript framework for zk-SNARKs and zkApps",
-  "version": "0.12.2",
+  "version": "0.12.3",
```

Comment:
should this be a major bump since its not backwards compatible with already compiled contract/programs (verification key changes)?

Reply:
Done.